docs/install.md

```diff
@@ -439,6 +446,23 @@ Features currently in "alpha" are:
 | [Object Params and Results](pipelineruns.md#specifying-parameters)                                                               | [TEP-0075](https://github.com/tektoncd/community/blob/main/teps/0075-object-param-and-result-types.md)                  |                [v0.38.0](https://github.com/tektoncd/pipeline/releases/tag/v0.38.0)                                                |                             |
 | [Array Results](pipelineruns.md#specifying-parameters)                                                               |            [TEP-0076](https://github.com/tektoncd/community/blob/main/teps/0076-array-result-types.md)       |       [v0.38.0](https://github.com/tektoncd/pipeline/releases/tag/v0.38.0)                                                           |                |
 
+### Beta Features
+
+Beta features are fields of stable CRDs that follow our "beta" [compatibility policy](../api_compatibility_policy.md).
+To enable these features, set the `enable-api-fields` feature flag to `"beta"` in
+the `feature-flags` ConfigMap alongside your Tekton Pipelines deployment:
+
+```sh
+kubectl edit configmap feature-flags -n tekton-pipelines
+```
+
+For beta versions of Tekton CRDs, setting `enable-api-fields` to "beta" is the same as setting it to "stable".
```

Comment:
One thing that might be confusing here is that these docs don't distinguish between fields that are in beta because they're newly introduced, and fields that are in beta because they were in v1beta1 but we don't want to stabilize them. Open to suggestions on how best to document this.